plugins/modules/system/listen_ports_facts.py

```diff
@@ -220,18 +299,34 @@ def getPidUser(pid):
     }
 
     try:
-        netstat_cmd = module.get_bin_path('netstat', True)
+        command = None
+        bin_path = None
+        if module.params['command'] is not None:
+            command = module.params['command']
+            bin_path = module.get_bin_path(command, required=True)
+        else:
+            for c in commands_map.keys():
```

Comment:
```suggestion
            for c in sorted(commands_map):
```
Dictionary order is not fixed, so you cannot rely on it. (It is only fixed for Python 3.5 or 3.6 and later; I forgot which of the two.)

Reply:
@felixfontein 
Good catch, thank you!